drop-ins/object-cache.php

```diff
@@ -150,9 +152,28 @@ function wp_cache_incr( $key, $offset = 1, $group = '' ) {
 function wp_cache_init() {
     global $wp_object_cache;
 
+    if ( ! defined( 'WP_REDIS_PREFIX' ) && getenv( 'SPINUPWP_CACHE_KEY_SALT' ) ) {
+        define( 'WP_REDIS_PREFIX', getenv( 'SPINUPWP_CACHE_KEY_SALT' ) );
+    }
+
+    if ( ! defined( 'WP_REDIS_SELECTIVE_FLUSH' ) && getenv( 'WP_REDIS_SELECTIVE_FLUSH' ) ) {
+        define( 'WP_REDIS_SELECTIVE_FLUSH', (bool) getenv( 'WP_REDIS_SELECTIVE_FLUSH' ) );
+    }
+
+    // Backwards compatibility: map `WP_CACHE_KEY_SALT` constant to `WP_REDIS_PREFIX`.
+    if ( defined( 'WP_CACHE_KEY_SALT' ) && ! defined( 'WP_REDIS_PREFIX' ) ) {
+        define( 'WP_REDIS_PREFIX', WP_CACHE_KEY_SALT );
+    }
```

Comment:
`WP_REDIS_PREFIX` and `WP_CACHE_KEY_SALT` needs reworking to handle all of these possibilities:

https://github.com/deliciousbrains/spinupwp-plugin/blob/develop/drop-ins/object-cache.php#L2001